server/views/pages/contacts/create/enterDob.njk

```diff
@@ -0,0 +1,111 @@
+                {% set dobHtml %}
+                    {{ govukDateInput({
+                        id: "dob",
+                        fieldset: {
+                            legend: {
+                                text: "Date of birth",
+                                classes: "govuk-visually-hidden"
+                            }
+                        },
+                        hint: {
+                            text: "For example, 15 6 2000"
+                        },
+                        items: [
+                            {
+                                id: 'day',
+                                label: 'Day',
+                                name: "day",
+                                classes: 'govuk-input--width-2',
+                                value: formResponses.day
+                            },
+                            {
+                                id: 'month',
+                                label: 'Month',
+                                name: "month",
+                                classes: 'govuk-input--width-2',
+                                value: formResponses.month
+                            },
+                            {
+                                id: 'year',
+                                label: 'Year',
+                                name: "year",
+                                classes: 'govuk-input--width-4',
+                                value: formResponses.year
+                            }
+                        ]
+                    }) }}
+
+                {% endset %}
+                {{ govukRadios({
+                    name: "isDobKnown",
+                    items: [
+                        {
+                            value: true,
+                            text: "Yes",
+                            checked: formResponses.isDobKnown === 'true',
+                            conditional: {
+                                html: dobHtml
+                            }
+                        },
+                        {
+                            value: false,
+                            text: "No",
+                            checked: formResponses.isDobKnown === 'false'
+                        }
+                    ],
+                    errorMessage: validationErrors | findError('isDobKnown')
+                }) }}
+
+                <div class="govuk-button-group">
+                    {{ govukButton({
+                        html: "Continue",
+                        type: "submit",
+                        classes: 'govuk-!-margin-top-6',
+                        attributes: {"data-qa": "continue-button"}
+                    }) }}
+                    <a class="govuk-link govuk-link--no-visited-state govuk-!-font-size-19" href="/">Cancel</a>
+                </div>
+            </form>
+
+        </div>
+    </div>
+
+{% endblock %}
+{% block pageScripts %}
+    <script src="/js/card.js"></script>
+{% endblock %}
```

Comment:
Does it need the card.js on this page? Didn't see a card on it?

Reply:
nope, will tidy up and imports. good spot